### ISSUE TITLE
Docs: Position of Explanations FAQ

### ISSUE THREAD

**Issue by DanielVandH** [MakieOrg/Makie.jl#4284]
The https://docs.makie.org/stable/explanations/faq page appears alphabetically when you hover over the Explanations tab. I wonder if for an FAQ it makes sense for it to be in its own spot rather than sorting it with everything else? e.g.

![image](https://github.com/user-attachments/assets/403f908c-10f7-4ce7-b95c-7494cbb422bb)

(or wherever is appropriate). Just my opinion anyway, maybe it's fine as is.

**Comment by asinghvi17** [comment]
Agreed, we could also have it be its own section.  This is a pretty easy edit I think (might just be editing `docs/src/.vitepress/config.mts`)
